semgrep/semgrep/config_resolver.py

```diff
@@ -485,13 +488,17 @@ def generate_config(fd: IO, lang: Optional[str], pattern: Optional[str]) -> None
 
 
 def get_config(
-    pattern: str, lang: str, config_strs: List[str]
+    pattern: str, lang: str, config_strs: List[str], autofix: Optional[str] = None
```

Comment:
Can we make these names consistent? It seems like `fix_expression` is the one we're using in cli